dali/operators/reader/loader/video/frames_decoder_test.cc

```diff
@@ -36,6 +36,7 @@ class FramesDecoderTestBase : public VideoTestBase {
     ASSERT_EQ(decoder.Width(), ground_truth.Width());
     ASSERT_EQ(decoder.Channels(), ground_truth.NumChannels());
     ASSERT_EQ(decoder.NumFrames(), ground_truth.NumFrames());
+    ASSERT_EQ(decoder.IsVfr(), ground_truth.IsVfr());
```

Comment:
Can you add a test for a case with non-vfr video file?

Reply:
This test covers both VFR and CFR files.

Reply:
Ok. Missed that.